js/impress.js

```diff
@@ -724,24 +724,31 @@
         //   Well, the [tab] key by default navigates around focusable elements, so clicking
         //   it very often caused scrolling to focused element and breaking impress.js
         //   positioning. I didn't want to just prevent this default action, so I used [tab]
-        //   as another way to moving to next step... And yes, I know that for the sake of
-        //   consistency I should add [shift+tab] as opposite action...
+        //   as another way to moving to next step...
         document.addEventListener( "keyup", function( event ) {
 
+            // Handles Tab and Shift + Tab
+            if ( event.keyCode === 9 ) {
+              if ( event.shiftKey ) {
+                api.prev();
+              } else {
+                api.next();
+              }
+              return;
+            }
+
             if ( event.shiftKey || event.altKey || event.ctrlKey || event.metaKey ) {
```

Comment:
I could find [the commit](https://github.com/FagnerMartinsBrack/impress.js/commit/d8995aa3194f6fdbcc8867626d52f913d0c52882), but not the original issue that describes the bug those lines fix.

Do you remember which browser and OS the problem happens? In Chrome on Mac, the event is not triggered when switching windows using <kbd>command</kbd> + <kbd>Tab</kbd>.

The only thing that is triggered is the following:

```shel
> event.keyCode: 91
> event.shiftKey: false
```

The code 91 represents OSLeft, which in Mac is the <kbd>command</kbd> key.